### ISSUE TITLE
Accession numbers in database tutorial are misleading

### ISSUE THREAD

**Issue by rsharris** [sourmash-bio/sourmash#1785]
This is more of an FYI than a bug.

The accession numbers shown in the database tutorial at
https://sourmash.readthedocs.io/en/latest/tutorial-basic.html#make-and-search-a-database-quickly
are misleading. It is easy to mistake them for the accession numbers of the sequences being compared. But what was actually compared were assemblies in which the accessions shown are simply the name of the first sequence in the assembly.

I was trying to reproduce that example, including generating the signatures from fasta (because I intended to compare sourmash distances to a different distance metric). I downloaded the accession numbers that are shown, and was surprised to find that the signatures I was generating didn't match the ones provided for the tutorial.

Eventually (by snooping inside the provided .sig files) I realized that, e.g., NZ_JHDG01000001.1.sig represents *all* the sequences in GCF_000601135.1, not just the first sequence.

**Comment by ctb** [comment]
thanks @rsharris you are indeed correct, of course! That's a legacy of me being ~lazy and naming the signatures in the genome databases after the name of the first contig in the respective FASTA file, way back when!

We've corrected this in our newer databases, but never went back and fixed the tutorial. Thanks for filing this issue to get it on our radar! Much appreciated!

**Comment by ctb** [comment]
Incidentally, in re distance metrics and (presumably :) error bounds, you might be interested in this paper - https://dib-lab.github.io/2020-paper-sourmash-gather/ - and a partner paper from David Koslicki's lab. They're just being posted to bioRxiv today; ping me if you'd like a link to the other one when it's up.

**Comment by rsharris** [comment]
Thanks. David is in fact one of the collaborators on the project I’m currently working on (you may have guessed that). Please do ping me.

Bob H

**Comment by ctb** [comment]
hi @rsharris here you go - [Debiasing FracMinHash and deriving confidence intervals for mutation rates across a wide range of evolutionary distances](https://www.biorxiv.org/content/10.1101/2022.01.11.475870v2)

**Comment by rsharris** [comment]
@ctb Danke!

**Comment by ctb** [comment]
and @taylorreiter just pointed me at yours 😆 https://www.biorxiv.org/content/10.1101/2022.01.14.476226v1

**Comment by rsharris** [comment]
Yep. It turns out that the tutorial-related experiment I was going to do ended up not being feasible in the short timeframe I had (but not due to any issue with sourmash), and so there's no sign of it in that paper.
